docs/source/reference/storage.rst

```diff
@@ -1,201 +1,131 @@
 .. _sky-storage:
 
-SkyPilot Storage
-=================
+Accessing Object Stores
```

Comment:
Cloud Object Storage / Cloud Buckets?

Reply:
+1 for this as well. Would it be possible to do `Cloud Buckets (Object Storage)`?

Reply:
Changed to `Using Object Storages`